cmd/tailscale/cli/serve.go

```diff
@@ -160,7 +160,14 @@ type localServeClient interface {
 // It also contains the flags, as registered with newServeCommand.
 type serveEnv struct {
 	// flags
-	json bool // output JSON (status only for now)
+	json             bool   // output JSON (status only for now)
```

Comment:
Might be a little ugly but we can potentially best this under another struct just distinguish v1 params from v2 ones. Or add a comment at the top